canonical-web-and-design/maas-ui
# Enable copying SSL keys

Done:
- Make the clipboard button copy the SSL key. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/237.

QA:
- View the SSL list in your preferences.
- Click the clipboard button next to a SSL key.
- Paste your clipboard preferences somewhere, it should contain the full SSL key.
